src/Swr/swrText.c

```diff
@@ -84,13 +84,62 @@ char* swrText_Translate(char* text)
     return NULL;
 }
 
-// 0x004503e0
+// 0x0049eb80
+int swrText_Sprintf(char *str, const char *format, ...) {
```

Comment:
This should just be sprintf as its the stdlibrary function defined in swr_stdlib.h - will update when the other patch is up allowing for calling standard library functions

Reply:
It's already defined in `swr_stdlib.h` yes, like most of the static stdlib. I don't get what you want to do exactly ?

Reply:
Yeah sorry I made this comment after I did the commit, as I realized exactly what you said - that I should have just called sprintf. I didn't look at swr_stdlib.h until after I figured out this was sprintf

I will put up another patch just using sprintf. The other patch I mentioned was this one https://github.com/tim-tim707/SW_RACER_RE/pull/18 but sprintf is one stdlib function that doesn't crash if you use the msvcrt ones (unlike the file ones which do crash)

Reply:
I have updated this to just use sprintf instead. This should be good to merge now